Makefile

```
@@ -0,0 +1,163 @@
+
+prover:
+	tmux kill-session -t p; \
+	tmux new -d -s p; \
+	tmux send-keys -t p "cd ${ZKSYNC_PROVER_HOME}/prover && export ZKSYNC_HOME=${ZKSYNC_PROVER_HOME}" Enter; \
+	tmux send-keys -t p "../bin/zk f env FRI_PROVER_SETUP_DATA_PATH=${ZKSYNC_PROVER_HOME}/prover/vk_setup_data_generator_server_fri/data cargo run --features "gpu" --release --bin zksync_prover_fri" Enter;
+
+compressor:
+	tmux kill-session -t c; \
+	tmux new -d -s c; \
+	tmux send-keys -t c "cd ${ZKSYNC_PROVER_HOME}/prover && export ZKSYNC_HOME=${ZKSYNC_PROVER_HOME}" Enter; \
+	tmux send-keys -t c "../bin/zk f cargo run --release --bin zksync_proof_fri_compressor" Enter;
+
+explorer:
+	tmux kill-session -t e; \
+	tmux new -d -s e; \
+	tmux send-keys -t e "cd ${ZKSYNC_PROVER_HOME}/prover && export ZKSYNC_HOME=${ZKSYNC_PROVER_HOME}" Enter; \
+	tmux send-keys -t e "cd ${ZKSYNC_EXPLORER_HOME}" Enter; \
+	tmux send-keys -t e "npm run dev" Enter;
+
+portal:
+	tmux kill-session -t po; \
+	tmux new -d -s po; \
+	tmux send-keys -t po "cd ${ZKSYNC_PORTAL_HOME}" Enter; \
+	tmux send-keys -t po "echo y | npx serve .output/public/ -p 3002" Enter;
+
+# Setup
+
+download-portal:
+	git clone https://github.com/matter-labs/dapp-portal.git ${ZKSYNC_PORTAL_HOME} || exit 0
+	cp diffs/portal.diff ${ZKSYNC_PORTAL_HOME}
+	git -C ${ZKSYNC_PORTAL_HOME} apply portal.diff || exit 0
+
+download-explorer:
+	git clone https://github.com/matter-labs/block-explorer.git ${ZKSYNC_EXPLORER_HOME} || exit 0
+	cp diffs/explorer.diff ${ZKSYNC_EXPLORER_HOME}
+	git -C ${ZKSYNC_EXPLORER_HOME} apply explorer.diff || exit 0
+
+download-core:
+	git -C ${ZKSYNC_CORE_HOME} checkout ${CORE_COMMIT} && git -C ${ZKSYNC_CORE_HOME} pull origin ${CORE_COMMIT} --ff-only || git clone https://github.com/matter-labs/zksync-era.git ${ZKSYNC_CORE_HOME} -b ${CORE_COMMIT}
+	cp diffs/observability.diff ${ZKSYNC_CORE_HOME}
+	git -C ${ZKSYNC_CORE_HOME} apply observability.diff || exit 0
+	cp ${ZKSYNC_ENV}.toml ${ZKSYNC_CORE_HOME}/etc/env/configs/
+
+download-prover:
+	git -C ${ZKSYNC_PROVER_HOME} checkout ${PROVER_COMMIT} && git -C ${ZKSYNC_PROVER_HOME} pull origin ${PROVER_COMMIT} --ff-only || git clone https://github.com/matter-labs/zksync-era.git ${ZKSYNC_PROVER_HOME} -b ${PROVER_COMMIT}
+	cp ${ZKSYNC_ENV}.toml ${ZKSYNC_PROVER_HOME}/etc/env/configs/
+
+download: download-core download-prover download-portal download-explorer
+
+init-explorer:
+	cd ${ZKSYNC_EXPLORER_HOME} && \
+	export ZKSYNC_HOME=${ZKSYNC_CORE_HOME} && \
+	npm i && \
+	npm run db:create && \
+	npm run build
+
+init-portal:
+	cd ${ZKSYNC_PORTAL_HOME} && \
+	export ZKSYNC_HOME=${ZKSYNC_CORE_HOME} && \
+	npm i && \
+	npm run generate:node:shyft
+
+init-core:
+	sudo chown -R admin:admin /home/admin/
+	cd ${ZKSYNC_CORE_HOME} && \
+	export ZKSYNC_HOME=${ZKSYNC_CORE_HOME} && \
+	export PATH=${ZKSYNC_CORE_HOME}/bin:$(PATH) && \
+	export PATH=/snap/bin:$(PATH) && \
```

```suggestion
	export PATH=${ZKSYNC_CORE_HOME}/bin:$$PATH && \
	export PATH=/snap/bin:$$PATH && \
```
It doesn't work because the second export "shadows" the first export, the suggestion solves the problem.

---

Fixed